plugins/modules/win_ntp.ps1

```diff
@@ -0,0 +1,189 @@
+#!powershell
+
+# Copyright: (c) 2020, Sakar Mehra <sakarmehra100@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$spec = @{
+  options = @{
+      peerlist = @{ type= "list"; elements = "str" }
+      enabled = @{ type= "bool"; default= "True" }
+      time_zone = @{ type= "str"; default= "UTC" }
+      factory_default = @{ type= "bool"; default= "False" }
+      type = @{ type= "str"; choices = "NTP","NT5DS","NoSync","AllSync"; }
+      cross_site_sync_flags = @{ type= "str"; choices = "0","1","2" }
```

Comment:
Have the flags be human readable. I shouldn't have to look at the docs to figure out what `cross_site_sync_flags: 0` means. The module can convert our human readable value to the relevant integer and not have the user worry about it at all. See https://github.com/ansible-collections/community.windows/pull/150/files#r515439307.